### ISSUE TITLE
build issue

### ISSUE THREAD

**Issue by dsadinoff** [hebcal/hebcal#113]
Haven't built from scratch in a while.  We seem to have an autoconf issue (at least on centos 6 and ubuntu 11.0.4)

3.11 doesn't have the problem, but at first glance it looks like most of the most recent github releases do:
````
configure...yaddayadda...

dsadinoff@danny:~/hebcal-3.18$ make
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/bash /home/dsadinoff/hebcal-3.18/missing aclocal-1.15
/home/dsadinoff/hebcal-3.18/missing: line 81: aclocal-1.15: command not found
WARNING: 'aclocal-1.15' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'aclocal' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
make: *** [aclocal.m4] Error 127
````

I'll try to chase it down when I have a moment

**Comment by dsadinoff** [comment]
also 4.0.1

**Comment by mjradwin** [comment]
You can try doing this as a workaround?

aclocal && autoconf && automake --foreign --add-missing && ./configure && make

I don't know why the generated configure script that I added to the repo is broken.  But I will try to reproduce because people shouldn't have to do all of the autocorrect nonsense. 

--
Sent from my mobile device

On Nov 3, 2015, at 10:14 PM, dsadinoff <notifications@github.com> wrote:

also 4.0.1

**Comment by mjradwin** [comment]
I published v4.0.2 and tested building on CentOS 6.7. Fix seemed to be to disable "maintainer mode". Give it a whirl and let me know if it works for you?

**Comment by dsadinoff** [comment]
Looks good.  Autoconf is such a maze of twisty passages.   Can anyone hope to use it without googling it's mysteries?

Well done.
